lib/ast-converter.js

```diff
@@ -256,6 +256,26 @@ function fixExports(node, result, ast) {
 }
 
 /**
+ * Returns true if a given TSNode is a token
+ * @param  {TSNode} node the TSNode
+ * @returns {boolean}     is a token
+ */
+function isToken(node) {
+     /**
+     * Note: We treat JsxText like a token (TypeScript doesn't classify it as one),
```

Comment:
@DanielRosenwasser just FYI this does indeed work as of TS 2.2!

I am removing this extra check in the PR to support 2.2 -> #169